src/start.sh

```diff
@@ -75,10 +75,9 @@ start() {
         sleep 0.5
     done
 
-    #  Wait until the FTL log contains the "FTL started" message before continuing
-    while ! grep -q '########## FTL started' /var/log/pihole/FTL.log; do
-        sleep 0.5
-    done
+    #  Wait until the FTL log contains the "FTL started" message before continuing, timeout after 10 seconds
+    logsize_before=$(stat -c%s /var/log/pihole/FTL.log)
+    pihole-FTL wait-for '########## FTL started' /var/log/pihole/FTL.log 10 "${logsize_before}" > /dev/null
```

Comment:
_Something_, and I don't know what, is giving me pause on the 10 second timeout here. 

While it _shouldn't_ take anywhere near 10 seconds for FTL to start up, if (for whatever reason) it _doesn't_ start up then the script will continue on as though it _has_. Currently we go around forever and ever and ever and ever - so it's an improvement for sure - but maybe this logic can be shored up a little

Reply:
We could exit if the return code is != 0 indicating that is has not been found